apps/site/assets/js/support-form.js

```diff
@@ -364,7 +364,7 @@ const validators = {
   },
   vehicle: function($) {
     const value = $("#vehicle").val();
-    return /^[0-9]*$/.test(value);
+    return value.length == 0 || /^[0-9]{1,8}$/.test(value);
```

Comment:
suggestion (non-blocking): I think it would also work to use `{0,8}` which would mean not needing the length check.